src/core/orderbook.js

```diff
@@ -448,11 +449,10 @@ OrderBook.prototype.applyTransferRate = function(balance) {
   assert(!isNaN(balance), 'Balance is invalid');
   assertValidNumber(this._issuerTransferRate, 'Transfer rate is invalid');
 
-  const adjustedBalance = OrderBookUtils.normalizeAmount(balance)
-  .divide(this._issuerTransferRate)
-  .multiply(Amount.from_json(OrderBook.DEFAULT_TRANSFER_RATE))
-  .to_json()
-  .value;
+  const adjustedBalance = (new Value_IOU(balance))
+  .divide(new Value_IOU(this._issuerTransferRate))
+  .multiply(new Value_IOU(OrderBook.DEFAULT_TRANSFER_RATE))
```

Comment:
Should probably just make _issuerTransferRate and OrderBook.DEFAULT_TRANSFER_RATE IOValues on assignment